src/content/cloud/okteto-cli.mdx

```diff
@@ -83,10 +95,17 @@ For example, for the image `registry.cloud.okteto.net/cindy/hello-world:okteto`,
 - `OKTETO_BUILD_HELLO_WORLD_REPOSITORY`: cindy/hello-world
 - `OKTETO_BUILD_HELLO_WORLD_SHA`: okteto@sha256:xxx
 
-### Built-in Tools When Deploying To Okteto Cloud
+In order to refer to endpoints of [external resources](reference/manifest.mdx/#external-object-optional) you can use the following environment variables:
+- `OKTETO_EXTERNAL_<external>_ENDPOINTS_<endpoint>_URL`: the URL of an endpoint of an external resource.
+
+In order to refer to endpoints/variables of [dependencies](reference/manifest.mdx/#dependencies-object-optional) you can use the following environment variables:
+- `OKTETO_DEPENDENCY_<dependency>_VARIABLE_<variable>`: the value of a variable of a dependency.
+- `OKTETO_DEPENDENCY_<dependency>_EXTERNAL_<external>_ENDPOINTS_<endpoint>_URL`: the URL of an endpoint of an external resource.
+
+### Built-in Tools When Deploying To Okteto
 
-When launching a development environment [from a Git Repository](cloud/deploy-from-git.mdx), Okteto will run an installation job that clones the Git repository, checks out the selected branch, builds the images defined in the `build` section of your Okteto Manifest, and executes the sequence of `deploy` commands. The job will fail if any of the commands in the `deploy` list fails.
-The installation job will run in a Debian Linux container with the following tools preinstalled:
+When deploying your development environment, Okteto executes the sequence of `deploy` commands.
+The commands run in a Debian Linux container with the following tools preinstalled:
```

Comment:
This is only the case if the deploy.image is not defined right?

Reply:
It's mentioned a little later in this section, would you move it here?

Reply:
let's put it on both parts